week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -18,36 +18,59 @@ This means the order the songs are played in will be random, but each song will
 
  */
 
-
 class MusicPlayer {
-    // Add your code here
-
+  constructor() {
+    this.i = 0;
+    this.playList = [];
+  }
+  add(music, singer) {
+    this.playList.push(`${music} by ${singer}`);
+  }
+  play() {
+    if (this.i > this.playList.length || this.i < 0) {
+      console.log(`there is no song to play`);
+    } else {
+      console.log(`Currently Playing: ${this.playList[this.i]}`);
+    }
+  }
+  skip() {
+    if (this.i < this.playList.length - 1) {
+      this.i++;
+      this.play();
+    } else {
+      console.log(`There isn't any song to play`);
```

Comment:
I think it's fine to console log that there are no more songs to play, but another solution could be to start playing songs from the beginning